docs/source/guide/explanation/algorithms/anomaly/index.rst

```diff
@@ -1,10 +1,123 @@
-Anomaly Recognition
-===================
+Anomaly Detection
+=================
 
-.. toctree::
-   :maxdepth: 1
+Introduction
+************
+The datasets from real-world applications such as industrial, medical, and security are extremely unbalanced, with an abundance of normal images and a severe lack of abnormal samples. A second issue is that the definition and type of abnormality are constantly evolving, making it difficult to train a supervised model once and use it permanently.  An alternative approach is to train a model using only normal samples to learn normality.  During the validation or testing phases, a deviation from this would indicate an anomaly. The process of identifying such deviations or anomalies in data is known as anomaly detection.
 
+This section examines the solutions for anomaly detection offered by the OpenVINO Training Extensions library.
 
-   anomaly_classification
-   anomaly_detection
-   anomaly_segmentation
+
+Task Types
+**********
+OTX supports the following anomaly task types:
+
+* Anomaly Classification - (Image-level detection)
+* Anomaly Detection - (Box-level localization)
+* Anomaly Segmentation - (Pixel-level localization)
+
+.. note::
+   Only normal images are used to train all anomaly detection algorithms. The distinction between the tasks is the way in which the testing phase predicts or localises anomalies. In anomaly classification, for instance, anomalies are predicted at the image level; in anomaly detection, anomalies are localised using a bounding box; and in anomaly segmentation, anomalies are localised using a mask.
+
+
+Anomaly Classification
+----------------------
+Anomaly classification is the task of predicting normal and abnormal images at the image level. As noted above, a model is trained on only normal images. During the testing phase, the model predicts an anomaly score indicating the likelihood of an image being abnormal. The threshold for anomaly classification is either set by the user or adaptively tuned by OTX. An image is classified as abnormal if the anomaly score is above the threshold.
```

Comment:
~~OTX~~ OpenVINO Training Extensions

Reply:
Maybe we could provide a brief 1 sentence explanation how the adaptive threshold value is determined.